app/controllers/application_controller/ci_processing.rb

```diff
@@ -90,12 +90,12 @@ def build_ownership_info
     Rbac.filtered(User).each { |u| @users[u.name] = u.id.to_s }
     record = klass.find(@ownership_items[0])
     user = record.evm_owner if @ownership_items.length == 1
-    @user = user ? user.id.to_s : nil
+    @user = user.try(:id).to_s
```

Comment:
This will be slightly different because formerly `@user` could end up being nil, but after it will be an empty string.